### ISSUE TITLE
Getting Cannot find user defined WIQL query with a shared query specified

### ISSUE THREAD

**Issue by james1301** [rfennell/AzurePipelines#1872]
### Azure DevOps Extensions

Generate Release Notes (Node Cross Platform)

### Platform

Azure DevOps Services

### Extension Version

Latest

### Describe the bug

I am specifying wiqlSharedQueryName, that exists and is shared, and my pipeline agent user can see it. This is a self hosted pipeline agent.

But I am getting "Cannot find user defined WIQL query".

### Repo Steps

1. Use release notes task and specify a wiqlSharedQueryName to a shared query in your project
2. Run pipeline
3. Build output says "Cannot find user defined WIQL query"

### Expected Behavior

Should find my shared query

**Comment by rfennell** [comment]
Yes, it should work.  I need to retest as I see my automated test focus on inject WIQL not using shared queries.  I wonder if it is permission related?

**Comment by james1301** [comment]
Yeah, I'm getting this:

`Checking for user defined WIQL query
Getting Work Items using user defined WIQL query '/Shared Queries/MQC_Release_Notes_Query'
Cannot find user defined WIQL query '/Shared Queries/MQC_Release_Notes_Query'`

And permissions:

![image](https://github.com/rfennell/AzurePipelines/assets/17780517/488c3f85-931e-42b4-b973-c845876afcf8)

**Comment by james1301** [comment]
Found it thanks, searched for the right build service user and they didn't have the permission.
